1e/README.md

```diff
@@ -0,0 +1,50 @@
+# 1E
+
+## Overview
+
+The 1E Datadog integration enables you to automatically send 1E metrics to your Datadog account.
```

Comment:
We recommend providing more insight on the value of the integration and the integrations purpose